_posts/2021-11-16-release_0_5_x.md

```diff
@@ -71,7 +71,7 @@ load_and_normalize_pipeline(load_data(), normalize()).run()
 ```
 
 Our [quickstart](https://github.com/zenml-io/zenml/tree/main/examples/quickstart) and
-[functional guide](https://docs.zenml.io/v/0.5.7/guides/functional-api) are the perfect
+[functional guide](https://docs.zenml.io/) are the perfect
```

Comment:
There so no functional guide so I think we should rewrite this, maybe just call the link "docs" instead or sth

Reply:
Following the previous comments, as there is no functional guide anymore, I am removing this link as well.